tests/api/features/steps/common_steps.py

```diff
@@ -26,4 +26,10 @@ def step_impl(context, status_code):
 
 @then('I receive a response with the "{schema_name}" schema')
 def step_impl(context, schema_name):
-    validate_schema(context, schema_name)
\ No newline at end of file
+    validate_schema(context, schema_name)
+    
+@step('I check the data is')
+def step_impl(context):
+    expected_json = fill_payload(context, payload={})
+    for key, value in expected_json.items():
+        assert value==context.response[key],'expected value for {key} is {value},but actual is {context.response[key]}'
```

Comment:
If the idea is to send the actual "key" and "value" in the message, then it's missing a formatter:
```suggestion
        assert value == context.response[key], f'expected value for {key} is {value}, but actual is {context.response[key]}'
```